<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–£–º–Ω—ã–µ –∑–∞–º–µ—Ç–∫–∏</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>10</y>
      <width>591</width>
      <height>661</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>–°–ø–∏—Å–æ–∫ –∑–∞–º–µ—Ç–æ–∫</string>
    </property>
   </widget>
   <widget class="QListWidget" name="SpisokZametok">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>441</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddZametka">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>–°–æ–∑–¥–∞—Ç—å –∑–∞–º–µ—Ç–∫—É</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DelZametka">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>180</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>–£–¥–∞–ª–∏—Ç—å –∑–∞–º–µ—Ç–∫—É</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveZametka">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>240</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å –∑–∞–º–µ—Ç–∫—É</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>–°–ø–∏—Å–æ–∫ —Ç–µ–≥–æ–≤</string>
    </property>
   </widget>
   <widget class="QListWidget" name="SpisokZametok_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>441</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>–í–≤–µ–¥–∏—Ç–µ —Ç–µ–≥...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddZametka_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>–î–æ–±–∞–≤–∏—Ç—å –∫ –∑–∞–º–µ—Ç–∫–µ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DelZametka_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>–û—Ç–∫—Ä–µ–ø–∏—Ç—å –æ—Ç –∑–∞–º–µ—Ç–∫–∏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchZametka">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>572</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>üîé–ò—Å–∫–∞—Ç—å –∑–∞–º–µ—Ç–∫–∏ –ø–æ —Ç–µ–≥—Éüîé</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
